tests/all/limits.rs

```diff
@@ -354,7 +354,7 @@ fn test_pooling_allocator_initial_limits_exceeded() -> Result<()> {
     let mut pool = crate::small_pool_config();
     pool.total_memories(2)
         .max_memories_per_module(2)
-        .memory_pages(5)
+        .max_memory_size(5 * 65536)
```

Comment:
okay I am going to stop saying "ditto about named constants" now and assume you will go over the diff yourself to catch all of them at this point